mne/preprocessing/stim.py

```diff
@@ -4,10 +4,14 @@
 
 import numpy as np
 from scipy import signal, interpolate
+from ..evoked import Evoked
+from ..epochs import Epochs
+from ..utils import deprecated
 
 from .. import pick_types
 
 
+@deprecated('DEPRECATED: Use fix_stim_artifact_raw')
```

Comment:
Let's not use uppercase letters here. We don't do it in other places. And when you read it it feels like someone is yelling or shouting at you.